packages/graphql/src/graphql/objects/CreateInfo.ts

```diff
@@ -21,6 +21,7 @@ import { GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from "gr
 
 export const CreateInfo = new GraphQLObjectType({
     name: "CreateInfo",
+    description: "Information about the creation of a node or relationship.",
```

Comment:
I think a more accurate description would be "Information about the number of nodes and relationships created during a create mutation".